src/Unicorn/Standard Config Files/Unicorn.config

```diff
@@ -43,6 +43,9 @@
 			<defaults>
 				<!-- The source data store is where we read data from. Usually, this would be Sitecore. -->
 				<sourceDataStore type="Rainbow.Storage.Sc.SitecoreDataStore, Rainbow.Storage.Sc" singleInstance="true"/>
+				
```

Comment:
This belongs in a config patch file that is specific to Unicorn.TFS, as opposed to in Unicorn core; my guess is that we'd end up with two NuGet packages: Unicorn.TFS (which essentially is this config file and probably a sample configuration that wires TFS), and Rainbow.Storage.TFS (the patches to Rainbow). Make sense?